letsencrypt/plugins/standalone.py

```diff
@@ -198,9 +197,8 @@ def prepare(self):  # pylint: disable=missing-docstring
 
     def get_chall_pref(self, domain):
         # pylint: disable=unused-argument,missing-docstring
-        chall_pref = list(self.supported_challenges)
-        random.shuffle(chall_pref)  # 50% for each challenge
-        return chall_pref
+        return [chall for chall in SUPPORTED_CHALLENGES
```

Comment:
If the suggested change is made on `supported_challenges`, this function can simply be:
```
return self.supported_challenges
```